src/RTTComms.cpp

```diff
@@ -309,6 +309,7 @@ namespace BrainCloud
     void RTTComms::connect()
     {
 #if (TARGET_OS_WATCH != 1)
+        _disconnectedWithReason=false;
```

Comment:
Let's continue using the same coding guidelines as we always do in those files...
```
_disconnectedWithReason = false;
```